apps/api/src/services/external/apiNodeService.ts

```diff
@@ -36,26 +36,67 @@ export async function getChainStats() {
     60 * 5, // 5 minutes
     cacheKeys.getChainStats,
     async () => {
-      const bondedTokensQuery = axios.get<CosmosStakingPoolResponse>(`${apiNodeUrl}/cosmos/staking/v1beta1/pool`);
-      const supplyQuery = axios.get<CosmosBankSupplyResponse>(`${apiNodeUrl}/cosmos/bank/v1beta1/supply?pagination.limit=1000`);
-      const communityPoolQuery = axios.get<CosmosDistributionCommunityPoolResponse>(`${apiNodeUrl}/cosmos/distribution/v1beta1/community_pool`);
-      const inflationQuery = axios.get<CosmosMintInflationResponse>(`${apiNodeUrl}/cosmos/mint/v1beta1/inflation`);
-      const distributionQuery = axios.get<CosmosDistributionParamsResponse>(`${apiNodeUrl}/cosmos/distribution/v1beta1/params`);
-
-      const [bondedTokensResponse, supplyResponse, communityPoolResponse, inflationResponse, distributionResponse] = await Promise.all([
-        bondedTokensQuery,
-        supplyQuery,
-        communityPoolQuery,
-        inflationQuery,
-        distributionQuery
-      ]);
+      let bondedTokens;
+      let totalSupply;
+      let communityPool;
+      let inflation;
+      let communityTax;
+
+      try {
+        const bondedTokensQuery = await axios.get<CosmosStakingPoolResponse>(
+          `${apiNodeUrl}/cosmos/staking/v1beta1/pool`
+        );
+        bondedTokens = parseInt(bondedTokensQuery.data.pool.bonded_tokens);
+      } catch (error) {
+        console.error("Failed to fetch bonded tokens:", error);
+      }
```

Comment:
this call and below is really the same as in another file. Logger can be a parameter.

```typescript
const createLoggingExecutor = (logger: LoggerService) => async <T>(cb: () => Promise<T>, defaultValue?: T): Promise<T> => {
   try {
     return await cb();
   } catch (error) {
     logger.error({ event: `Failed to fetch ${cb.name}`, error });
     getSentry().captureException(error);
     return defaultValue;
   }
}

const logger = new LoggerService()
const runOrLog = createLoggingExecutor(logger)

const bondedTokens = await runOrLog(() => {
   const bondedTokensQuery = await axios.get<CosmosStakingPoolResponse>(
      `${apiNodeUrl}/cosmos/staking/v1beta1/pool`
   );
   return parseInt(bondedTokensQuery.data.pool.bonded_tokens);
})

// or
const bondedTokens = await runOrLog(
   () => axios.get<CosmosStakingPoolResponse>(`${apiNodeUrl}/cosmos/staking/v1beta1/pool`)
      .then(({ data }) => parseInt(data.pool.bonded_tokens))
)
```